### PR TITLE
Fix: install error

### PR DESCRIPTION
**Pull request by kakshay-ushur**
1. fixed bluebird install error 
2. added apiSecToken, entId support for notifyFwk.

### CODE REVIEW

File: lib/Ushur.js [hunk 1]
```diff
@@ -126,11 +126,15 @@ Ushur.prototype.getResponses = function (params = {}) {
   var _this = this;
   var payload = {
     requestId: params.requestId,
-    cmd: params.requestId ? 'responseForReqId' : undefined,
+    cmd: params.cmd ? params.cmd : "responseForReqId",//support for cmd: "getCampaignList"
```
Comment (krupanand-shetty) [inline lib/Ushur.js]: is responseForReqId a valid cmd?

Comment (krupanand-shetty) [reply lib/Ushur.js]:
```suggestion
    cmd: params.cmd ? params.cmd : null,
```

File: lib/Ushur.js [hunk 2]
```diff
@@ -126,18 +128,22 @@ Ushur.prototype.getResponses = function (params = {}) {
   var _this = this;
   var payload = {
     requestId: params.requestId,
-    cmd: params.requestId ? 'responseForReqId' : undefined,
+    cmd: params.requestId ? "responseForReqId" : params.cmd,
```
Comment (krupanand-shetty) [inline lib/Ushur.js:131]: what is this change?

Comment (kakshay-ushur) [reply lib/Ushur.js:131]: earlier cmd value was ignored(used **undefined** instead) even though it was sent in the request.

File: lib/Ushur.js [hunk 3]
```diff
@@ -33,9 +32,12 @@ function Ushur(credentials) {
    * @type {String}
    */
   this.tokenId = undefined;
-
+  this.hostname = credentials.hostname || null;
   //Authenticate in the start
-  this.auth();
+  if(credentials.mode && credentials.mode.toLowerCase() == "promise")
```
Comment (krupanand-shetty) [inline lib/Ushur.js:37]: Can you also update readme regarding usage of hostname field with examples. And also usage of mode(promise)

**Pull request merged**